### ISSUE TITLE
New ID for Məlṭān zatənśāʾe (= Choir for the Resurrection")?

### ISSUE THREAD

**Issue by CarstenHoffmannMarburg** [BetaMasaheft/Documentation#2322]

**Comment by karljonaskarlsson** [comment]
No, I was not expressing myself clearly :)   I'm not sure that it is a good idea to start encoding individual antiphons in an unsystematic way. I think it's better to save that Sisyphus work for another time, and simply encode what you, Carsten, have in the manuscript as "Service for Easter beginning with mǝlṭān za-tǝnśāʾe" and give a long incipit going from the beginning and at least up to እንተ፡ ርኢናሃ፡ ለእኪት። (or even longer, as the text was already typed). I.e., to onlyu create one work ID
